doc/book/replication/repl_leader_elect.rst

```diff
@@ -69,10 +69,15 @@ Eventually, a leader is elected.
 
 All the non-leader nodes are called *followers*. The nodes that start a new
 election round are called *candidates*. The elected leader sends heartbeats to
-the non-leader nodes to let them know it is alive. So if there are no heartbeats
-for a period set by the :ref:`replication_timeout <cfg_replication-replication_timeout>`
-option, a new election starts. Terms and votes are persisted by
-each instance in order to preserve certain Raft guarantees.
+the non-leader nodes to let them know it is alive.
+
+In case there are no heartbeats for a period set by the :ref:`replication_timeout <cfg_replication-replication_timeout>` option,
```

Comment:
An election is started if there are no heartbeat for `4 * replication_timeout`. This value is denoted `replication_death_timeout` internally, but it is not a configuration option, and I don't know if it's mentioned anywhere in the doc or not.

Reply:
Corrected.